darkside-tests/src/utils.rs

```diff
@@ -674,10 +674,12 @@ pub mod scenarios {
                 DarksideSender::IndexedClient(n) => self.get_lightclient(n),
                 DarksideSender::ExternalClient(lc) => lc,
             };
-            lightclient
-                .send_from_send_inputs(vec![(receiver_address, value, None)])
-                .await
-                .unwrap();
+            zingo_testutils::lightclient::from_inputs::send(
```

Comment:
I like your creativity to move lightclient test methods into zingo-testutils in terms of code organisation and simplification, although I think including the full path on every send/shield looks awful and is crazy. I am very happy in test writing to actually just write `send` with `use::zingo_testutils::from_inputs::send` imported. that will actually clean up our test code rather than turn `recipient.send` into a monster on many lines of test code

Reply:
I agree with you.   I like:

```Rust
use::zingo_testutils::from_inputs;
...
from_inputs::FOO(spam...
```

I am aiming to consistently apply that pattern moving forward.